<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1740</width>
    <height>977</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="CurrentPlayer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>CurrentPlayer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="UpcomingPlayerCombo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>UpcomingPlayer: UpcomingPlayerScore</string>
    </property>
   </widget>
   <widget class="QLabel" name="UpcomingPlayerAfterCombo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>UpcomingPlayer: UpcomingPlayerScore</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainWindowButtonAddPlayer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainWindowButtonRemovePlayer">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainWindowButtonPlayerManager">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Player Manager</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainWindowButtonEditPlayer">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="darts_ShotsRemaining">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>SHOTS REMAINING</string>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentScore">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>CurrentScore</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="RoundScoreLabel">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>0</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Round Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoundScoreCumulated">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>0</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RoundScore</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoundScoreFirstLabel">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1.:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoundScoreSecondLabel">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2.:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoundScoreThirdLabel">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3.:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoundScoreFirstThrow">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>FirstThrowScore</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoundScoreSecondThrow">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>SecondThrowScore</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoundScoreThirdThrow">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ThirdThrowScore</string>
    </property>
   </widget>
   <widget class="Line" name="CurrentPlayerStatusLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>16</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>5</pointsize>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="MainWindowButtonRevertAction">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>UNDO</string>
    </property>
   </widget>
   <widget class="QLabel" name="dartboardView">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>150</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Shot1Indicator">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>0</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Shot2Indicator">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>0</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Shot3Indicator">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>0</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1740</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menubar_Game">
    <property name="title">
     <string>Game</string>
    </property>
    <widget class="QMenu" name="menubar_Game_menuGame_Modes">
     <property name="title">
      <string>Game Modes</string>
     </property>
     <addaction name="menuGame_Modes_action301"/>
     <addaction name="menuGame_Modes_action501"/>
     <addaction name="menuGame_Modes_action701"/>
     <addaction name="separator"/>
     <addaction name="menuGame_Modes_actionSingle_Out"/>
     <addaction name="menuGame_Modes_actionDouble_Out"/>
     <addaction name="menuGame_Modes_actionMaster_Out"/>
    </widget>
    <addaction name="menubar_Game_actionRestart"/>
    <addaction name="menubar_Game_actionLoad"/>
    <addaction name="menubar_Game_actionSave"/>
    <addaction name="menubar_Game_actionSaveAs"/>
    <addaction name="menubar_Game_actionManualMode"/>
    <addaction name="separator"/>
    <addaction name="menubar_Game_menuGame_Modes"/>
   </widget>
   <widget class="QMenu" name="menubar_Settings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="menubar_Settings_actionUndo"/>
    <addaction name="menubar_Settings_actionRedo"/>
    <addaction name="menubar_Settings_actionLanguage"/>
    <addaction name="separator"/>
    <addaction name="menubar_Settings_actionPlayerManager"/>
    <addaction name="separator"/>
    <addaction name="menubar_Settings_actionPlayerPlaceholder"/>
   </widget>
   <widget class="QMenu" name="menubarHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menubarSpacer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QMenu" name="menubarDebugWindows">
    <property name="title">
     <string>DebugWindow</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QMenu" name="menubarDevTools">
    <property name="title">
     <string>DevTools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menubarTestTrigger">
    <property name="title">
     <string>TestTrigger</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSwitchPalette">
    <property name="title">
     <string>Dark Mode</string>
    </property>
   </widget>
   <addaction name="menubar_Game"/>
   <addaction name="menubar_Settings"/>
   <addaction name="menubarHelp"/>
   <addaction name="menubarSpacer"/>
   <addaction name="menubarDebugWindows"/>
   <addaction name="menu"/>
   <addaction name="menubarDevTools"/>
   <addaction name="menubarTestTrigger"/>
   <addaction name="menuSwitchPalette"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menubar_Game_actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="menubar_Game_actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="menubar_Game_actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="menubar_Game_actionManualMode">
   <property name="text">
    <string>Manual Mode</string>
   </property>
  </action>
  <action name="menuGame_Modes_action301">
   <property name="text">
    <string>301</string>
   </property>
  </action>
  <action name="menuGame_Modes_action501">
   <property name="text">
    <string>501</string>
   </property>
  </action>
  <action name="menuGame_Modes_action701">
   <property name="text">
    <string>701</string>
   </property>
  </action>
  <action name="menuGame_Modes_actionSingle_Out">
   <property name="text">
    <string>Single Out</string>
   </property>
  </action>
  <action name="menuGame_Modes_actionDouble_Out">
   <property name="text">
    <string>Double Out</string>
   </property>
  </action>
  <action name="menuGame_Modes_actionMaster_Out">
   <property name="text">
    <string>Master Out</string>
   </property>
  </action>
  <action name="menubar_Settings_actionUndo">
   <property name="text">
    <string>Undo...</string>
   </property>
  </action>
  <action name="menubar_Settings_actionRedo">
   <property name="text">
    <string>Redo...</string>
   </property>
  </action>
  <action name="menubar_Settings_actionPlayerManager">
   <property name="text">
    <string>Player Manager</string>
   </property>
  </action>
  <action name="menubar_Settings_actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="menubar_Settings_actionPlayerPlaceholder">
   <property name="text">
    <string>$Player.Placeholder</string>
   </property>
  </action>
  <action name="menubar_Game_actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
